src/ledger/database/rocksdb.zig

```diff
@@ -331,7 +332,7 @@ pub fn RocksDB(comptime column_families: []const ColumnFamily) type {
     };
 }
 
-fn callRocks(logger: Logger, comptime func: anytype, args: anytype) ReturnType(@TypeOf(func)) {
+fn callRocks(logger: anytype, comptime func: anytype, args: anytype) ReturnType(@TypeOf(func)) {
```

Comment:
This should use the same scope as RocksDB.

Reply:
I think it does. Since the `callRocks` function is called from within the `RocksDB ` type function, and the logger scoped within `RocksDB` is passed to it. 

Just that, since this is being called from the `RocksDB` type function, I could not find a way to properly typed the logger in `callRocks` but to use anytype.

Reply:
Oh I see. I didn't read this carefully enough.

It might be best to move callRocks into the RocksDB struct, so it can utilize the generic type without resorting to anytype.

Reply:
Actually, I think it would probably be better to set the scope manually to "rocksdb" for the entire file. As is, the scope is going to be pretty weird, because it will include every column family. I think the string will be huge and messy.

Reply:
[cae2755](https://github.com/Syndica/sig/pull/345/commits/cae27554ca6894d0d8a05e56f7baa8b499b5bf05)